src/Features/Core/Portable/Completion/CompletionOptions.cs

```diff
@@ -29,8 +29,7 @@ internal sealed record class CompletionOptions
         public bool UpdateImportCompletionCacheInBackground { get; init; } = false;
         public bool FilterOutOfScopeLocals { get; init; } = true;
         public bool ShowXmlDocCommentCompletion { get; init; } = true;
-        public bool? ShowNewSnippetExperience { get; init; } = null;
-        public bool SnippetCompletion { get; init; } = true;
+        public bool ShowNewSnippetExperience { get; init; } = false;
```

Comment:
nit: since this isn't set by anyone now 
`public bool ShowNewSnippetExperience => false;`

Reply:
It's used by the tests